### PR TITLE
add heading component tests, improve tests using getKnobsInput() method

### PR DESCRIPTION
**Pull request by marcinstachowicz**
## Automation tests for Storybook components

**Added to testing scope:**
- Heading

**Improvements:**
- using getKnobsInput() method

### CODE REVIEW

File: cypress/integration/heading.feature [hunk 1]
```diff
@@ -0,0 +1,105 @@
+Feature: Heading component
+  I want to change Heading component properties
+
+  Background: Open Heading component component page
```
Comment (DlgSHi) [inline cypress/integration/heading.feature]: one component word is unnecessary, Heading component it's called Heading

File: cypress/locators/index.js [hunk 2]
```diff
@@ -1,29 +1,23 @@
...
+  ENABLE_BACKGROUND_UI_CHECKBOX, HEIGHT_INPUT, INPUT_WIDTH_SLIDER,
+  LABEL_WIDTH_SLIDER, LABEL_INLINE_CHECKBOX, LABEL_ALIGN_SELECT,
   HELP_ICON_PREVIEW, FIELD_HELP_PREVIEW, LABEL_HELP_PREVIEW, FORM, STORY_ROOT,
   SHOW_CLOSE_ICON_CHECKBOX, STICKY_FORM_FOOTER, DISABLE_ESC_KEY_CHECKBOX,
-  CLOSE_ICON_BUTTON, BACKGROUND_UI_LOCATOR,
+  CLOSE_ICON_BUTTON, BACKGROUND_UI_LOCATOR, LINK,
 } from './locators';
 
 // knobs locators
+export const getKnobsInput = propertyName => cy.get(`[name="${propertyName}"]`);
```
Comment (DlgSHi) [inline cypress/locators/index.js]: locator should be moved to locators.js file

Comment (DlgSHi) [reply cypress/locators/index.js]: export const KNOBS_NAME = propertyName => `[name="${propertyName}"]`; 

i potem użyć w index.js
export const getKnobsInput = propertyName => cy.get(KNOBS_NAME(propertyName));

File: cypress/integration/heading.feature (hunk 1 above)
Comment (marcinstachowicz) [reply cypress/integration/heading.feature]: fixed

File: cypress/locators/index.js (hunk 2 above)
Comment (marcinstachowicz) [reply cypress/locators/index.js]: resolved

**Pull request merged**